src/toil_scripts/gatk_germline/germline.py

```diff
@@ -490,81 +575,119 @@ def main():
 
-        if config['run_vqsr']:
+        if inputs['run_vqsr']:
             vqsr_fields = {'phase', 'mills', 'dbsnp', 'hapmap', 'omni'}
-            require(config_fields > vqsr_fields,
+            require(input_fields > vqsr_fields,
                     'Missing parameters for VQSR:\n{}'
-                    .format(', '.join(vqsr_fields - config_fields)))
+                    .format(', '.join(vqsr_fields - input_fields)))
+
+        # Set resource parameters
+        inputs['xmx'] = convert_space_resource(inputs['xmx'], '10G')
+        inputs['file_size'] = convert_space_resource(inputs['file_size'], '10G')
+        inputs['cores'] = inputs.get('cores', 8)
 
         # GATK recommended annotations:
         # https://software.broadinstitute.org/gatk/documentation/article?id=2805
-        config['annotations'] = ['QualByDepth', 'FisherStrand', 'StrandOddsRatio',
-                                 'ReadPosRankSumTest', 'MappingQualityRankSumTest',
-                                 'RMSMappingQuality', 'InbreedingCoeff']
+        inputs['annotations'] = ['QualByDepth',
+                                 'FisherStrand',
+                                 'StrandOddsRatio',
+                                 'ReadPosRankSumTest',
+                                 'MappingQualityRankSumTest',
+                                 'RMSMappingQuality',
+                                 'InbreedingCoeff']
+
+        # Login to Synapse using the Python API
+        if inputs.get('synapse_credentials', None) is not None:
+            synapse_name, synapse_pwd = open(inputs['synapse_credentials'],
+                                             'r').readline().strip().split(',')
+            inputs['synapse_login'] = synapseclient.login(synapse_name,
+                                                          synapse_pwd,
+                                                          rememberMe=False,
+                                                          silent=True)
+        else:
+            inputs['synapse_login'] = None
 
-        # It is a convention to store configuration attributes in a Namespace object
-        config = argparse.Namespace(**config)
+        # It is a toil-scripts convention to store input parameters in a Namespace object
+        config = argparse.Namespace(**inputs)
 
+        # Download shared files and start germline pipeline
         shared_files = Job.wrapJobFn(download_shared_files, config).encapsulate()
-        run_pipeline = Job.wrapJobFn(run_gatk_germline_pipeline,samples, shared_files.rv())
+        run_pipeline = Job.wrapJobFn(run_gatk_germline_pipeline,
+                                     samples,
+                                     shared_files.rv()).encapsulate()
         shared_files.addChild(run_pipeline)
 
+        if inputs['run_oncotator']:
+            require(inputs['assembly'].lower() == 'hg19',
+                    'Check your reference assembly. We recommend using hg19 when using Oncotator')
```

Comment:
1. Please don't lock support to a single reference build. That type of hardcoding has cost literal weeks of my life. If I can save someone else that pain, I will have not died in vain.
2. If you are going to lock support to a single reference build, provide a flag to override it and document it well.
3. If you're throwing an exception, use stronger verbiage than "recommend".